### DISCUSSION TITLE
Copy Paste Multiple Items in Tile Inspector

### DISCUSSION THREAD

**Discussion by xp4xbox** [OpenRCT2/OpenRCT2#15707] [Feature requests]
It would be really nice to be able to copy and paste multiple items in tile inspector. Especially for repetitive building.

**Comment by harrysbird** [comment]
I Love this idea, I would be nice to add this feature

**Comment by Liampie** [comment]
Copy+pasting entire tiles would be hot.

**Comment by Nubbie** [comment]
Related to #1926

**Comment by J-PIE-314** [comment]
The ability to copy and paste multiple items has been implemented in this plug-in: https://github.com/Sadret/openrct2-scenery-manager
